examples/chatgpt/rag-quickstart/pinecone-retool/gpt-action-pinecone-retool-rag.ipynb

```diff
@@ -0,0 +1,491 @@
4577999, 0.004243878647685051, 0.010716047137975693, -0.02044624462723732, 0.000830966280773282, 0.015665549784898758, -0.0002823344257194549, 0.022782571613788605, 0.01712406799197197, -0.012363630346953869, -0.0010550613515079021, 0.030115671455860138, 0.0016627766890451312, -0.0014720215694978833, -0.010857847519218922, 0.0053613996133208275, -0.011620867066085339, 0.021337559446692467, 0.010878104716539383, 0.028008924797177315, 0.011046914383769035, -0.0037577063776552677, 0.011924725025892258, 0.025942692533135414, 0.000775681110098958, -0.00670850183814764, -0.01341700367629528, 0.009912512265145779, -0.00024203106295317411, -0.002132067922502756, -0.014828253537416458, -0.007090012077242136, 0.007501908112317324, 0.0011943294666707516, 0.007090012077242136, 0.012073277495801449, -0.012309611774981022, 0.003717191983014345, 0.009298044256865978, -0.00014728655514772981, -0.013059127144515514, 0.004058187827467918, 0.010108331218361855, -0.0028089953120797873, -0.01038517989218235, -0.024524688720703125, 0.005985995754599571, -0.005425547249615192, -0.006465415935963392, 0.01754271611571312, 0.010689036920666695, 0.031196054071187973, -0.0041054547764360905, 0.01677294261753559, -0.025497034192085266, 0.023390287533402443, -0.01693500019609928, 0.0048279608599841595, -0.01110768597573042, 0.01573307439684868, 0.00033529847860336304, 0.004409312270581722, -0.011046914383769035, -0.015260406769812107, 0.0025152661837637424, -0.02386295422911644, 0.0021253156010061502, 0.022552991285920143, 0.0075289178639650345, -0.02286360040307045, -0.01138453371822834, 0.0028511977288872004, 0.006880688015371561, 0.006137924734503031, 0.006131172180175781, 0.025361986830830574, -0.014423110522329807, 0.029494451358914375, -0.02357935532927513, 0.0030925958417356014, -0.013761376030743122, -0.005077799316495657, 0.01270800270140171, -0.01520638819783926, -0.0012517247814685106, -0.00828518532216549, -0.0031685600988566875, -0.007515412755310535, -0.007657213136553764, 0.006131172180175781, -0.004054811783134937, -0.012620221823453903, 0.0065160589292645454, 0.016759438440203667, 0.040622394531965256, -0.007643708493560553, -0.011195466853678226, -0.0253484807908535, -0.010250131599605083, 0.02467324212193489, -0.004979889374226332, -0.0064350301399827, 0.02002759650349617, -0.005077799316495657, -0.02960248850286007, -0.004780693911015987, 0.006056895945221186, -0.011553343385457993, -0.002032470190897584, 0.004358669277280569, -0.013673595152795315, 0.004608507733792067, -0.016340790316462517, -0.021999293938279152, -0.009716692380607128, -0.02892724983394146, -0.021067464724183083, 0.015260406769812107, -0.015476483851671219, -0.008913158439099789, -0.007218307815492153, 0.009797721169888973, -0.01428806222975254, -0.04775292053818703, 0.017920849844813347, 0.010378426872193813, 0.012890317477285862, -0.012680993415415287, -0.014355586841702461, -0.004118959419429302, 0.003457224927842617, 0.0008904717396944761, -0.009399330243468285, -0.00203247019089758
```

Comment:
any chance we can both black out URL (subdomain at least) and also have the GPT looking in the action without prompting it to (so it always search out for the information) ?